### PR TITLE
Elite Mercenary Non-Deathsquad Version

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Alt to: https://github.com/cmss13-devs/cmss13/pull/6740

Adds a new non-Deathsquad varient of the Elite Mercenary ERT. 

These mercenaries lack Deathsquad tier armour and Whiteout weapons. Their armour is now akin to Medium Marine armour but with Light movement speed, and they use mostly common or UPP style weapons. 

# Explain why it's good for the game

Opens up the opportunity for these rarely used ERTs to be used more often for admin events. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: The Elite Mercenary emergency response team has been split into a Deathsquad and non-Deathsquad version. Previously this ERT only functioned with Deathsquad weapons/armour. 
admin: The Elite Mercenary Deathsquad version can be denoted with the title of (!DEATHSQUAD!). The non-Deathsquad version possess weapons and armour similar to a typical Marine, albeit they still possess night vision optics. 
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/gear_presets/_select_equipment.dm [hunk 1]
```diff
@@ -662,6 +662,36 @@ GLOBAL_LIST_INIT(rebel_rifles, list(
 				new_human.equip_to_slot_or_del(new /obj/item/storage/belt/marine, WEAR_WAIST)
 		spawn_weapon(gunpath, ammopath, M, 0, ammo_amount)
 
+/datum/equipment_preset/proc/spawn_merc_elite_weapon_lowthreat(atom/M, ammo_amount = 12, shotgun_chance = 50, spawn_belt = 1)
```
Comment (harryob) [inline code/modules/gear_presets/_select_equipment.dm:665]: why is this defined so high up instead of on `/datum/equipment_preset/other/elite_merc`

Comment (harryob) [reply code/modules/gear_presets/_select_equipment.dm:665]: also given you're only piping humans into this, you can just type it appropriately

File: code/modules/gear_presets/_select_equipment.dm [hunk 2]
```diff
@@ -662,6 +662,36 @@ GLOBAL_LIST_INIT(rebel_rifles, list(
...
+	var/list/elite_merc_shotguns = list(
+	/obj/item/weapon/gun/shotgun/merc = pick(GLOB.shotgun_handfuls_12g),
+	/obj/item/weapon/gun/shotgun/combat = pick(GLOB.shotgun_handfuls_12g),
+	/obj/item/weapon/gun/shotgun/type23 = pick(GLOB.shotgun_handfuls_8g))
+
+	if(prob(shotgun_chance))
+		var/gunpath = pick(elite_merc_shotguns)
+		var/ammopath = elite_merc_shotguns[gunpath]
+		if(spawn_belt)
+			if(ishuman(M))
```
Comment (harryob) [inline code/modules/gear_presets/_select_equipment.dm:681]: redundant, type is known

File: code/modules/gear_presets/_select_equipment.dm [hunk 3]
```diff
@@ -662,6 +662,36 @@ GLOBAL_LIST_INIT(rebel_rifles, list(
 				new_human.equip_to_slot_or_del(new /obj/item/storage/belt/marine, WEAR_WAIST)
 		spawn_weapon(gunpath, ammopath, M, 0, ammo_amount)
 
+/datum/equipment_preset/proc/spawn_merc_elite_weapon_lowthreat(atom/M, ammo_amount = 12, shotgun_chance = 50, spawn_belt = 1)
+	if(!M) return
```
Comment (harryob) [inline code/modules/gear_presets/_select_equipment.dm:666]: no inline control

**Review by harryob — CHANGES_REQUESTED** [review]

**Pull request closed without merging**